.github/workflows/ci.yaml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  go:
+    name: Check index-generator build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Go environment
+      uses: actions/setup-go@v2.1.2
+
```

Comment:
It would be better we specify the go version here (like below) as we use `go mod` and the lower version of go might not work properly

```yaml
with:
        # The Go version to download (if necessary) and use. Supports semver spec and ranges.
        go-version: 1.13
```

Reply:
Ooh, good catch.

Reply:
Updated